torch/_inductor/dtype_propagation.py

```diff
@@ -74,15 +73,6 @@ def promote_types(
     dtype_prop_candidates = []
 
     for arg in args:
-        if isinstance(arg, str):
-            # TODO: fix the flex attention instances, enable internally
-            if not config.is_fbcode():
```

Comment:
are there other fbcode issues this will pop now?

Reply:
Not sure, we'll see!